core/src/epicli/data/common/ansible/playbooks/roles/sshtunnel/tasks/setup.yml

```diff
@@ -0,0 +1,8 @@
+
+---
+- include_tasks: "teardown.yml"
```

Comment:
We would to have names for all tasks since Ansible linter requires them by default.